recipes/pycgns/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pyCGNS" %}
+{% set version = "5.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 949210707a2dd9da36cb397daf83b5d8a0b07a4fa46628383b84e2c66840a9dd
+
+build:
+  number: 0
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python
+    - hdf5   
+    - numpy
+    - cython
+    - sphinx
+    - future
+    - pkgconfig
+    - setuptools
+
+  host:
+    - python
+    - hdf5
+    - numpy
+    - cython
+    - future
+    - pkgconfig
+    - setuptools
+    - pip
+
+  run:
+    - python
+    - hdf5
+    - numpy
```

Comment:
If you build against numpy, then you need to use a pin_compatible statement here. See our documentation.